content/tutorials/map_algebra/basic_map_algebra.qmd

```diff
@@ -0,0 +1,585 @@
+---
+title: "Basics of map algebra"
+author: "Brendan Harmon"
+date: 2025-06-18
+date-modified: today
+categories: [map algebra, raster, terrain, beginner]
+description: >
+  Learn the basics of map algebra in GRASS
+  including arithmetic, functions, and logic. 
+  Use map algebra to model a landscape.
+thumbnail: images/map_algebra_00.webp
+format:
+  ipynb: default
+  html:
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+engine: jupyter
+execute:
+  eval: false
+jupyter: python3
+---
+
+![Terrain modeled with map algebra](images/map_algebra_01.webp)
+
+::: {.callout-note title="Computational notebook"}
+This tutorial can be run as a 
+[computational notebook](https://grass-tutorials.osgeo.org/content/tutorials/map_algebra/basic_map_algebra.ipynb).
+Learn how to work with notebooks with the tutorial
+[Get started with GRASS & Python in Jupyter Notebooks](../get_started/fast_track_grass_and_python.qmd).
+:::
+
+# Introduction
+This tutorial is a gentle introduction 
+to the basics of map algebra in GRASS. 
+You will begin to learn how to think procedurally 
+about space using math, logic, and flow control. 
+Map algebra is an algebraic approach
+to modeling fields of data such as rasters.
+It can be used to create, summarize, transform, 
+or combine multiple rasters. 
+Algebraic operations on raster maps can be classified 
+as local, focal, zonal, or global.
+Local operations act on each raster cell in isolation.
+Focal operations act on a neighborhood around each cell.
+Zonal operations act on groups of cells. 
+Global operations act on all cells in the raster.
+
+::: {#tbl-panel layout-ncol=1}
+| Types  | Scope                          |
+|--------|--------------------------------|
+| Local  | Each cell individually         |
+| Focal  | Each cell and its neighborhood |
+| Zonal  | Groups of cells                |
+| Global | All cells                      |
```

Comment:
This could be used as a way to point users to other tools, e.g. local (r.mapcalc), focal (r.neighbors, rmfilter, r.mapcalc), zonal (r.stats.zonal, r.univar, v.rast.stats, r.mapcalc), global(r.univar)